alf/utils/dist_utils.py

```diff
@@ -251,6 +251,47 @@ def _compute_log_prob(single_distribution, single_action):
     return total_log_probs
 
 
+def sample_action_distribution(nested_distributions):
```

Comment:
IF this is a general function API, suggest adding a flag for `rsample` if reparameterized is needed

Reply:
```rsample``` option added